target/linux/sunxi/image/cortexa7.mk

```diff
@@ -47,6 +47,14 @@ define Device/friendlyarm_nanopi-neo-air
 endef
 TARGET_DEVICES += friendlyarm_nanopi-neo-air
 
+define Device/friendlyarm_zeropi
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi ZeroPi
```

Comment:
Just ZeroPi?

Reply:
Good spotting, yes it is just ZeroPi, no nanopi